package.json

```diff
@@ -11,5 +11,6 @@
   "devDependencies": {
     "gulp": "^4.0.0",
     "gulp-sass": "^4.0.0"
-  }
+  },
```

Comment:
Can you completely undo the changes to this file?

Reply:
I've not manually changed this file at all, any changes to it are the result of npm. Is it back to how it was now?